tests/scripts/analyze_outcomes.py

```diff
@@ -166,7 +166,7 @@ def analyze_driver_vs_reference(results: Results, outcomes: Outcomes,
                     ignored = True
 
         if not ignored and not suite_case in driver_outcomes.successes:
-            results.error("PASS -> SKIP/FAIL: {}", suite_case)
+            results.error("SKIP/FAIL -> PASS: {}", suite_case)
```

Comment:
FWIW, I find the new message confusing. In my mental model,  `a -> b` means "when we go from built-in to driver, things change from a to b", because in my mind the direction of the change is "from built-in to driver".

Clearly the 3 people who made and approved this change didn't read the same thing as me in this arrow. And I can see how having it ordered the other way in the header was confusing.

I'm now thinking it would be better to be more explicit, and avoid relying on comparison with the header (which in the code is in a different place) or reading too much into simple arrow. So, something like "ref: PASS but driver: SKIP/FAIL" would probably be better.

(As discussed elsewhere with Gilles, we could also avoid some noise by ignoring failures and focusing only on whether the test was skipped or not, in which case we'd have "ref: PASS/FAIL but driver: SKIP" or "ref: not SKIP but driver: SKIP".)

Anyway, something explicit and self-contained (no reliance on a possibly-distant header, even less on someone's implicit mental model of which direction the arrow means).

Reply:
I have the same mental model that naturally favors `reference -> driver`, but for me, the order in the header trumps this: given a basic mental model that says “reference first, then driver”, and text just above that says “driver” first, then ”reference”, the text above wins. The obvious way to reconcile the two would be to change the order in `driver … vs reference …`, but being more explicit on each line is good anyway.